### PR TITLE
cleanup API report warnings in some more packages

### PR DESCRIPTION
**Pull request by Rugvip**
🧹🧹🧹🧹🧹🧹🧹

### CODE REVIEW

File: packages/backend-test-utils/src/database/TestDatabases.ts [hunk 1]
```diff
@@ -113,7 +115,7 @@ export class TestDatabases {
    * Returns a fresh, unique, empty logical database on an instance of the
    * given database ID platform.
    *
-   * @param id The ID of the database platform to use, e.g. 'POSTGRES_13'
+   * @param id - The ID of the database platform to use, e.g. 'POSTGRES_13'
```
Comment (freben) [inline packages/backend-test-utils/src/database/TestDatabases.ts:118]: Not a fan of the dashes honestly, is this a preferred format?

File: packages/catalog-model/src/validation/entityKindSchemaValidator.ts [hunk 2]
```diff
@@ -22,12 +22,13 @@ import { compileAjvSchema, throwAjvError } from './ajv';
...
  * schema apiVersion/kind didn't apply to that data, or throws a
- * {@link TypeError} describing actual errors.
+ * {@link globals#TypeError} describing actual errors.
  *
  * This validator is highly specialized, in that it has special treatment of
  * the `kind` and `apiVersion` root keys. This only works if your schema has
  * their rule set to `"enum"`:
  *
+ * ```
  * <pre>
```
Comment (freben) [inline packages/catalog-model/src/validation/entityKindSchemaValidator.ts:32]: should remove the pre then?

**Review by freben — APPROVED** [review]

File: packages/backend-test-utils/src/database/TestDatabases.ts (hunk 1 above)
Comment (freben) [reply packages/backend-test-utils/src/database/TestDatabases.ts:118]: It's almost like it takes the old pre-TS format where you were supposed to write a type there in curly braces but then leaving a dash instead kinda unnecessarily

Comment (Rugvip) [reply packages/backend-test-utils/src/database/TestDatabases.ts:118]: TSDoc says there has to be one, and api-extractor produces warnings if they're not there, so figured we might as well have them and not make a fuss 😅. I do remember having them in JSDoc too.

https://tsdoc.org/pages/tags/param/

**Review by jhaals — APPROVED** [review]

**Pull request merged**